.travis.yml

```diff
@@ -3,22 +3,27 @@ language: php
 matrix:
   include:
     - php: 5.3
+      env: COMPOSER_FLAGS="--prefer-lowest"
     - php: 5.4
     - php: 5.5
     - php: 5.6
       env: SCRUTINIZER_REPORT=true
     - php: 7.0
+      env: SYMFONY_VERSION='2.8.*'
+    - php: 7.0
+      env: SYMFONY_VERSION='3.0.*'
 
 cache:
   directories:
     - vendor
-    - $HOME/.composer/cache
+    - $HOME/.composer/cache/files
 
 before_install:
   - composer self-update
+  - if [ "$SYMFONY_VERSION" != "" ]; then composer require --dev --no-update symfony/symfony=$SYMFONY_VERSION; fi
 
 install:
-  - composer install --prefer-dist
+  - composer update $COMPOSER_FLAGS --prefer-dist --optimize-autoloader
```

Comment:
Sure!

The thing is, if you want to change the version of a package to backwards, you need to use composer update, and what's happening in this case composer is trying to get the lowest version of all packages to make sure compatibility is working fine. If you composer install it will just ignore any kind of change on requires and composer.json to install the composer.lock, which is not we want since now we are checking sf 2.1, 2.8 and 3.0 :)

That's why the update is required to work like that!
I did some tests locally and you can also check the build history inside travis, or I can show you in person if you want to!